src/components/ConfidentialQueuePanel.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { ListGroupItem } from 'reactstrap'
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faSpinner } from '@fortawesome/free-solid-svg-icons'
+
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
Any reason to not use a function component instead? I don't think we have any state here.